### DISCUSSION TITLE
[Bug]: What's PandoraNext? Why ChatGPT needs this?

### DISCUSSION THREAD

**Discussion by Originalimoc** [danny-avila/LibreChat#1235] [Troubleshooting]
### What happened?

When you use ChatGPT in UI, reply is: ```
Something went wrong. Here's the specific error message we encountered: Error message: "Failed to send message. HTTP 429 - {"detail":"It is recommended to upgrade to the latest PandoraNext: https://github.com/pandora-next/deploy"}"
```

### Steps to Reproduce

1. Set API key
2. Send a random message
3. error "reply"

### What browsers are you seeing the problem on?

Firefox, Chrome

### Relevant log output

_No response_

### Screenshots

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
See: https://discord.com/channels/1086345563026489514/1177595493610684536

The public reverse proxy isn’t working anymore. This is not an officially supported method and we would sooner remove it than fix it (as it’s against ChatGPT ToS)

The public one is configured to be used by default

**Comment by Originalimoc** [comment]
So only deploy a private one?

**Reply by danny-avila** [reply]
not that simple. you need to get around cloudflare, and the last public working method was archived 2 months ago

https://github.com/acheong08/ChatGPTProxy

**Reply by danny-avila** [reply]
actually, it looks like the public one was just using pandoranext, it might be possible to deploy your own to use chatgpt with librechat: https://github.com/pandora-next/deploy

**Comment by fuegovic** [comment]
Here's the guide to deploy your own:
https://docs.librechat.ai/features/pandoranext.html

Be warned that this will only give you access to the free model in LibreChat. GPT Plus models and feature will not work.
